### PR TITLE
Pull Request for Issue1843: Creating fast shutter implementation

### PR DESCRIPTION
**Pull request by helfrij**
Created new class BioXASFastShutter, a subclass of AMExclusiveStatesEnumeratedControl. The class allows someone to set the status control (determines the current state) and the operator control (allows for movement between states). Added instance to BioXASSideBeamline, and a view in BioXASSidePersistentView.

### CODE REVIEW

File: source/beamline/BioXAS/BioXASFastShutter.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			clearStates();
+
+		operator_ = newOperator;
+
+		if (operator_) {
+			addState(0, "Open", 0, operator_, 1);
+			addState(1, "Closed", 1, operator_, 0);
+		}
+
+		qDebug() << "\n\n" << toString();
```
Comment (dretrex) [inline source/beamline/BioXAS/BioXASFastShutter.cpp]: Probably don't need this debug anymore.

**Comment by dretrex** [comment]
Looks good!
Time Estimate: 2 minutes

File: source/beamline/BioXAS/BioXASFastShutter.cpp (hunk 1 above)
Comment (helfrij) [inline source/beamline/BioXAS/BioXASFastShutter.cpp]: Yup, you're right. Removed.

**Comment by helfrij** [comment]
Thanks @dretrex.

**Pull request merged**